gitlint-core/setup.py

```diff
@@ -60,12 +60,15 @@ def get_version(package):
     ],
     python_requires=">=3.6",
     install_requires=[
-        'Click==8.0.3',
-        'arrow==1.2.1',
+        'Click>=8',
+        'arrow>=1',
```

Comment:
This seems unnecessarily wide, why not `>=1.2` Same for `sh>=1.14`

Reply:
Gitlint uses a tiny subset of the arrow API that works as far back as 0.3.2.

I’ll go ahead and bound `sh>=1.13.0` since `_unify_ttys` is used in the `qa` tests—even though we still run tests with pinned requirements, and the program itself works fine as far back as sh 1.05, the earliest version supporting Python 3.3+.